front-end/src/app/shared/resolvers/transaction.resolver.ts

```diff
@@ -79,7 +79,11 @@ export class TransactionResolver {
   }
 
   resolveExistingTransaction(transaction: Transaction): Observable<Transaction | undefined> {
-    if (transaction.children) {
+    if (
+      transaction.children &&
```

Comment:
Shouldn't this no longer include transaction.children in the if statement since it's no longer being passed from the backend. I just checked and it works ok because the frontend initializes it to an empty array, but it still doesn't feel like a needed check here. Also I just checked with a Conduit earmark receipt and ran into issues because the type was ScheduleATransactionTypes.CONDUIT_EARMARK_RECEIPT_DEPOSITED which was not included in the MultipleEntryTransactionTypes

Reply:
Awesome! Thanks.